code/game/machinery/vending.dm

```diff
@@ -1107,14 +1107,14 @@
 					  /obj/item/storage/fancy/cigcartons/homeless = 1,
 					)
 	prices = list(/obj/item/clothing/mask/smokable/cigarette/cigar = 200,
-				  /obj/item/storage/fancy/cigarettes = 100,
+				  /obj/item/storage/fancy/cigarettes = 150,
 				  /obj/item/storage/fancy/cigcartons = 800,
 				  /obj/item/storage/box/matches = 10,
 				  /obj/item/flame/lighter/random = 5,
-				  /obj/item/storage/fancy/cigar = 450,
-				  /obj/item/storage/fancy/cigarettes/killthroat = 100,
+				  /obj/item/storage/fancy/cigar = 1000,
+				  /obj/item/storage/fancy/cigarettes/killthroat = 200,
```

Comment:
I'm surprised the cigarettes with literal poison in them are more expensive than the normal ones